docs/wiki/configurator/vtx-tab.md

```diff
@@ -25,7 +25,7 @@ It is important to have VTX Telemetry (SmartAudio or Tramp) enabled in the Ports
 | **Power**                    | Select the power level to use.                                                                                                                                                                                                                                                                                                                                                      |
 | **Pit Mode**                 | When enabled, the VTX enters in a very low power mode to let the quad be on at the bench without disturbing other pilots. Usually the range of this mode is less than 5m.<br/>NOTE: Some protocols, like SmartAudio, can't enable Pit Mode via software after power-up.                                                                                                             |
```

Comment:
it is unclear here what are the implications of the Note in here.

- does it mean I cannot enable pit mode from the configurator if I use SmartAudio?
- how do I enable PitMode to be on when I power my craft?

Reply:
- correct, SmartAudio cannot enable pitmode via configurator.
- i suppose it depends on what hardware you have.  maybe smart-audio does not have true pirmode, but rather only a lowest setting -- i am unsure and defer to some racers to answer.  @limonspb ?